schema/dataKeys.json

```diff
@@ -422,7 +432,6 @@
                     "type": "array",
                     "items": {
                       "type": "object",
-                      "additionalProperties": false,
```

Comment:
What's the motivation for this change? I don't know what's allowed for `contentGateway`, but I guess there was a reason to not allow additional properties?

Reply:
It was down to extra params being added that are not user provided).
[Slack Thread here](https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1731934464613269?thread_ts=1731665042.843589&cid=C04PZ7H0VA8)

Reply:
Ah, ok, so it was based on a discussion, then it's good.